bciers/apps/registration/app/components/operations/registration/FacilityInformationPage.tsx

```diff
@@ -54,7 +54,7 @@ const FacilityInformationPage = async ({
   return (
     <FacilityInformationForm
       facilityId={facilityId}
-      isCreating={!isOperationSfo && !facilityId}
+      isCreating={!isOperationSfo || !facilityId}
```

Comment:
Unrelated fix I am sneaking in here. I introduced this bug which will cause an error if you pick an SFO operation on page 1, page 2 will have an error on submission if there is a facility tied to that one since it will POST instead of PUT.